core/arch/arm/kernel/core_bootcfg.c

```diff
@@ -169,7 +169,7 @@ static struct map_area bootcfg_memory_map[] = {
 };
 
 /*
- * bootcfg_get_pbuf_is_handler - return the platform specfic pbuf_is
+ * bootcfg_get_pbuf_is_handler - return the platform specific pbuf_is
```

Comment:
I can do that. Or what about putting these files in `core/arch/arm/boot` instead? I have no strong preference.